examples/org.eclipse.jface.text.examples/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: CodeMinig Examples
 Bundle-SymbolicName: org.eclipse.jface.text.examples
-Bundle-Version: 1.2.0.qualifier
+Bundle-Version: 1.2.1.qualifier
```

Comment:
i think this has to be. 1.2.100

Reply:
thanks, should be corrected with the last upload